src/lib/json-rpc-client.js

```diff
@@ -1,24 +1,18 @@
-// TODO this is duplicated!
+import { JSONRPCError, isNotification, isResponse } from './json-rpc.js'
```

Comment:
I needed to copy/paste these files over as the build system is funky about importing from the web-extension folder. I couldn't easily change it and I didn't want to delay this PR, so I will address this in another PR.